hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -157,6 +197,26 @@ public boolean enabled() {
     return getBoolean(ENABLE);
   }
 
+  public boolean isMetadataIndexBloomFilterEnabled() {
```

Comment:
Since this is MetadataConfig, it is implied that all of these are metadata related. So you may drop the word "Metadata" from these function names to shorten them.

Reply:
fixed.